docs/installation/windows.md

```diff
@@ -14,10 +14,14 @@ There is a shortcut placed in the Start Menu upon installation.
 
 ### Prerequisites
 
+To enable PowerShell remoting over WinRM, the following prerequisites need to be met:
+
 * Install the [Universal C Runtime](https://www.microsoft.com/download/details.aspx?id=50410) on Windows versions prior to Windows 10.
   It is available via direct download or Windows Update.
   Fully patched (including optional packages), supported systems will already have this installed.
-* Install the [Visual C++ Redistributable](https://www.microsoft.com/download/details.aspx?id=48145) for VS2015.
+* Install the Windows Management Framework (WMF) [4.0](https://www.microsoft.com/en-us/download/details.aspx?id=40855)
```

Comment:
The links should not contain the culture/language specific part `en-us`

Reply:
Good catch. Fixed.